stanford.profile

```diff
@@ -807,6 +807,8 @@ function stanford_system_info_alter(&$info, $file, $type) {
     preg_match("/Stanford Story Page/", $info['name']) ||
     preg_match("/Stanford Subsite/", $info['name']) ||
     preg_match("/Stanford Jumpstart/", $info['name']) ||
+    preg_match("/Stanford Sites Jumpstart/", $info['name']) ||
```

Comment:
If my memory serves me correctly we had this in before and took it out. It may have been because the modules were being disabled on form submission or something else. As this is an "any" match you may want to adjust the regular expression to "starts with".

Reply:
My memory is that we had `("Stanford Sites/")` and that was too inclusive. Let me find the PR.

Reply:
Just found it: https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/125

Reply:
Looks like it prevents the module from running and not just "hide"

Reply:
#125

Reply:
Perhaps I should update this PR with a tally of which particular modules it blocks.

Reply:
399 modules available before this change.

Reply:
389 available after this change.